mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultCard.tsx

```diff
@@ -86,27 +105,25 @@ export function VaultCard (props: VaultCardProps): JSX.Element {
                       {translate('components/VaultCard', 'None')}
                     </ThemedText>
                   )}
-                <TokenIconGroup symbols={props.collaterals} />
+                <TokenIconGroup symbols={props.collaterals.map(collateral => collateral.id)} maxIconToDisplay={3} />
               </View>
             </View>
           </View>
-          <View>
-            <ThemedIcon onPress={props.onArrowPress} iconType='MaterialIcons' name='chevron-right' size={20} style={tailwind('mt-1')} />
-          </View>
         </View>
         <View style={tailwind('flex flex-row flex-wrap -mb-2')}>
-          <VaultInfo label='Active loans' value={props.activeLoans} decimalPlace={0} />
-          <VaultInfo label='Total loan amount' value={props.totalLoanAmount} prefix='$' decimalPlace={2} />
-          <VaultInfo label='Collateral amount' value={props.collateralAmount} prefix='$' decimalPlace={2} />
+          <VaultInfo label='Active loans' tokens={props.activeLoans?.map(loan => loan.tokenId)} valueType='TOKEN_ICON_GROUP' />
+          <VaultInfo label='Total loan amount' value={props.totalLoanAmount} prefix='$' decimalPlace={2} valueType='NUMBER' />
```

Comment:
Hmm, it's supposed to be `Total loans value` based from the design if this is incorrect, please check the other labels as well e.g `Total Collaterals value`

Reply:
I have asked jonathan about this when i was developing, he mentioned the label should be the same with the UI in `Your vault`, however he hasn't update yet. I am keeping it the same as `Your vault` for now